### PR TITLE
add event filters

### PR DESCRIPTION
**Pull request by tmsdkeys**
Closes: #26 
This PR:

- adds events filters. Before we were listening for all IBC related events on dispatcher/universal channel handler
- styling fix wrt version

## Summary by CodeRabbit

- **New Features**
	- Introduced event filtering based on port address to enhance event management.
- **Improvements**
	- Updated event listener callbacks for better performance and accuracy.
	- Enhanced console log messages for clearer event tracking.
- **Refactor**
	- Minor code formatting for improved readability.

### CODE REVIEW

File: scripts/_events.js [hunk 1]
```diff
@@ -1,30 +1,38 @@
...
+const hre = require("hardhat");
+const path = require("path");
+
 const explorerOpUrl = "https://optimism-sepolia.blockscout.com/";
 const explorerBaseUrl = "https://base-sepolia.blockscout.com/";
+const configRelativePath = process.env.CONFIG_PATH || "config.json";
+const configPath = path.join(__dirname, "..", configRelativePath);
+const config = require(configPath);
+
+function filterChannelEvents(portAddress) {
```
Comment (tmsdkeys) [inline scripts/_events.js:10]: @kenobijon your original suggestion for channel events would have likely worked, but there's a chance someone manually works with the config in which case the sendPacket and createChannel fields do not necessarily equal one another

File: scripts/_events.js [hunk 2]
```diff
@@ -103,24 +106,33 @@ function listenForIbcChannelEvents (network, source, dispatcher) {
...
-        );
-
+          -------------------------------------------\n`);
+        }
         dispatcher.removeAllListeners();
     });
 }
 
-function listenForIbcPacketEvents (network, dispatcher) {
+function filterPacketEvents(portAddress, network) {
```
Comment (tmsdkeys) [inline scripts/_events.js:115]: @kenobijon your original filtering did not include universal packets. there's not difference in the event from IBC POV so we need to check both of the config fields (for now)

**Review by tmsdkeys — COMMENTED** [review]
@kenobijon please have a double check, I tested though and can confirm this works on my end

File: scripts/_events.js (hunk 1 above)
Comment (kenobijon) [reply scripts/_events.js:10]: that makes sense, using createChannel field explicitly seems best

**Review by kenobijon — APPROVED** [review]
Tested and all events are getting logged!

**Pull request merged**